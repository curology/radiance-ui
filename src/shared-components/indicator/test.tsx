import React from 'react';
import { render } from 'src/tests/testingLibraryHelpers';

import { Indicator } from './index';

describe('<Indicator />', () => {
  describe('UI snapshots', () => {
    it('renders the correct css with a text', () => {
      const { container } = render(<Indicator text="1" />);

      expect(container.firstElementChild).toMatchSnapshot();
    });

    it('renders the correct css with a number', () => {
      const { container } = render(<Indicator text={3} />);

      expect(container.firstElementChild).toMatchSnapshot();
    });
  });
});
