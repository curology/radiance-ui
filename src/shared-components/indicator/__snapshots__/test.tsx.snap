// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Indicator /> UI snapshots renders the correct css with a number 1`] = `
.emotion-0 {
  background-color: #BD200F;
  min-height: 16px;
  height: 16px;
  max-height: 16px;
  text-align: center;
  padding: 0 0.25rem;
  min-width: 16px;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 50%;
}

.emotion-0>div {
  color: #524D6E;
  font-size: 0.75rem;
  line-height: 1.67;
  font-weight: bold;
  color: #FFFFFF;
  -webkit-transform: translateY(1px);
  -moz-transform: translateY(1px);
  -ms-transform: translateY(1px);
  transform: translateY(1px);
}

<div
  class="emotion-0 emotion-1"
>
  <div>
    3
  </div>
</div>
`;

exports[`<Indicator /> UI snapshots renders the correct css with a text 1`] = `
.emotion-0 {
  background-color: #BD200F;
  min-height: 16px;
  height: 16px;
  max-height: 16px;
  text-align: center;
  padding: 0 0.25rem;
  min-width: 16px;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 50%;
}

.emotion-0>div {
  color: #524D6E;
  font-size: 0.75rem;
  line-height: 1.67;
  font-weight: bold;
  color: #FFFFFF;
  -webkit-transform: translateY(1px);
  -moz-transform: translateY(1px);
  -ms-transform: translateY(1px);
  transform: translateY(1px);
}

<div
  class="emotion-0 emotion-1"
>
  <div>
    1
  </div>
</div>
`;
